ci/run-docker.sh

```diff
@@ -19,7 +19,7 @@ run() {
       --env TARGET=$target \
       --env FEATURES=$2 \
       --env STDSIMD_TEST_EVERYTHING \
-      --volume `pwd`:/checkout:ro \
+      --volume `pwd`:/checkout \
```

Comment:
I did not managed to get this to work any other way :( 

The `checkout` directory is not readonly anymore because now building `stdsimd` requires building `coresimd` and the lock files are not generated a priori anymore.